### ISSUE TITLE
new netdata logo

### ISSUE THREAD

**Issue by ktsaou** [netdata/netdata#4476]
##### Feature idea summary

Replace the existing open-source logo with the new one developed by @gmosx 

1. provide new logos for every logo image found in https://github.com/netdata/netdata/tree/master/web/gui/images

2. change the logo in all third party sites:

   - [ ] github
   - [ ] twitter
   - [ ] facebook
   - [ ] CNCF
   - [ ] fossa
   - [ ] lgtm
   - [ ] travis
   - [ ] QBO
   - [ ] Docusign
   - [ ] slack
   - [ ] public netdata demo sites (will be automatically updated once this is merged into the repo, except the registry)

##### Expected behavior

The new logo appearing everywhere.

**Comment by SuperSandro2000** [comment]
Can you share a link to the new logo or a mock up of it? I would like to take a look at it

**Comment by ktsaou** [comment]
We will have the final version by Monday.

**Comment by ktsaou** [comment]
here it is:

![image](https://user-images.githubusercontent.com/2662304/47668573-7b5a2600-dbb1-11e8-8fb5-ed698c78c3fd.png)

**Comment by SuperSandro2000** [comment]
For me it does not look like it has anything todo with data

**Comment by gmosx** [comment]
It's not a strict requirement for the logo to have 'anything to do with data'. Also the logo is still being fine-tuned.

That being said, I am wondering if you have any specific ideas/suggestions on a possible logo for netdata?

**Comment by gmosx** [comment]
Also, the image uploaded was a bit rough... here is a better rendition of the logo (which is still fine-tuned)

![image](https://user-images.githubusercontent.com/37342/47716923-cbd29180-dc4c-11e8-8b8f-a81932d11135.png)

**Comment by SuperSandro2000** [comment]
This looks already better. Maybe I just need to get used to it

**Comment by gmosx** [comment]
![image](https://user-images.githubusercontent.com/37342/47802277-9acf8b00-dd38-11e8-9720-7bc5a1a17859.png)
